lib/wheel-indicator.js

```diff
@@ -201,17 +201,30 @@ var WheelIndicator = (function() {
         }
     }
 
+    // https://developer.mozilla.org/ru/docs/Web/API/EventTarget/addEventListener#Безопасная_проверка_поддержки_option
+    // https://developer.mozilla.org/ru/docs/Web/API/EventTarget/addEventListener#Улучшение_производительности_прокрутки_с_помощью_passive_true
+    var passiveSupported = false;
+    try {

```

Comment:
Better to wrap this code in the function like `isPassiveSupported` and put it somewhere at the bottom of file